src/vs/workbench/contrib/files/browser/fileActions.contribution.ts

```diff
@@ -61,6 +60,11 @@ if (isMacintosh) {
 // Commands
 CommandsRegistry.registerCommand('_files.windowOpen', openWindowCommand);
 CommandsRegistry.registerCommand('_files.newWindow', newWindowCommand);
+MenuRegistry.addCommand({
```

Comment:
@bpasero @sbatten 

I registered this command to MenuRegistry as it's part of `registerWorkbenchAction`, which consists of 

* `CommandsRegistry.registerCommand`
* `KeybindingsRegistry.registerKeybindingRule`
* `MenuRegistry.addCommand`
* `MenuRegistry.appendMenuItem`

I'm not 100% sure of the intention of holding a cache of commands in the MenuRegistry and `MenuRegistry.getCommand` is used in quite a few places.